crates/fluvio-spu-schema/src/server/fetch_offset.rs

```diff
@@ -24,9 +24,6 @@ use super::SpuServerApiKey;
 pub struct FetchOffsetsRequest {
     /// Each topic in the request.
     pub topics: Vec<FetchOffsetTopic>,
-
-    #[fluvio(min_version = 23)]
-    pub consumer_id: Option<String>,
```

Comment:
We should consider the compatibility effects of this change on older fluvio clients.  Can they send with this field and still interoperate?  Can the field be deprecated w/o deleting it?

Reply:
Make sense, I added it back with a deprecated attribute to keep working with older clients.